fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -2696,7 +2696,7 @@ private void toThriftForCommonParams(TExecPlanFragmentParams commonParams,
             commonParams.setProtocol_version(InternalServiceVersion.V1);
             commonParams.setFragment(fragment.toThrift());
             commonParams.setDesc_tbl(descTable);
-            commonParams.setFunc_version(3);
+            commonParams.setFunc_version(4);
```

Comment:
function_version is used for resolving aggregation/analytic function, so can we add another version?